optuna/trial.py

```diff
@@ -437,16 +523,22 @@ def set_user_attr(self, key, value):
 
         Example:
 
-            Save fixed hyperparameters of neural network training:
+            Save fixed hyperparameters of neural network training.
```

Comment:
This title is not consistent with the code. Is it possible to actually train the neural networks? Or I think we can work on it in the follow-up PR.